### DISCUSSION TITLE
SMTP providers - better support needed

### DISCUSSION THREAD

**Discussion by macwebcomputing** [ExpressionEngine/ExpressionEngine#4066] [Feature Requests]
**Description of the problem**
I can only use postmark for SMTP. Every other email provider services built for developers that I try are generating errors and fails. Like resend.com, mailersend or any. Tried 5 ones with different errors. 

**How To Reproduce**
Try SMTP settings from resend.com or mailersend
-->

**Error Messages**
"Message not ending with." often getting this but often no enough details. Also, unware if we need to choose SSL 1.2 .1.3 etc in most cases. This option is not provided by most providers. Should be automatically detected. 

**Screenshots / Videos / Template Code**
None

**Environment Details:**
 - Version: 7.3.14
 - PHP Version 8.1.17
 - MySQL Version 5..7.34
 - OS: macOS
 - Web Server: MAMP Pro

**Possible Solution**
Forced to use postmark :-) 

**Additional context**
Issues seems related to libs sending email in SMTP mode.

**Comment by chunkitdesign** [comment]
I recently experienced an issue since the system was upgraded to EE 7.3.15. Currently, I am using Mandrill, and it works. However, I am unsure if it is a server issue or an EE issue.

**Comment by intoeetive** [comment]
I have been able to successfully send email using `resend.com` by setting "Email Newline character" preference to "\r\n\"

I agree that the error message as returned by SMTP server gives no clue that you need to modify this setting though

Converting this issue into feature request and will look into possible ways to improve email sending
